docker-compose.yml

```diff
@@ -23,6 +24,9 @@ services:
       WAGTAILTRANSFER_SOURCE_BASE_URL: https://example.com/wagtail-transfer/
       WAGTAILTRANSFER_SOURCE_SECRET_KEY: wagtailtransfer-source-secret-key
       BASE_URL: 'http://localhost:8000'
+      CACHE_BACKEND: db
```

Comment:
We have a redis server available but we're using 'db' as the cache backend?

Reply:
we can use whichever cache backend we want to use or test here

Reply:
if we have decided to go with Redis or no-cache then I believe no-cache would be the default option

Reply:
used dummy cache as default option